templates/zerver/help/alert-words.md

```diff
@@ -0,0 +1,26 @@
+# Alert Words
+Zulip allows you to define custom alert words and phrases, which when said will notify you, similarly to being [mentioned](/help/at-mention-a-team-member) by another user.
+
+## Adding Alert Words
+1.  Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner of the right sidebar.
```

Comment:
I wouldn't call it the upper right corner of the sidebar, rather the cog in the navigation, but I might be wrong here.

Could you look for another example of this in already merged user docs? Being consistent here is key.